book_source/source/module-9-miscellaneous-topics/lesson-27-reading-ethics-in-data-science/case-study-4-facial-recognition.md

```diff
@@ -1,17 +1,13 @@
 # Case Study 4: Facial Recognition
 
-##  Task  
+## Task
 
-Read this [None](https://www.nytimes.com/2018/02/09/technology/facial-recognition-race-artificial-intelligence.html?module=inline) on the use of facial recognition in machine learning.  
+Read this [article](https://www.nytimes.com/2018/02/09/technology/facial-recognition-race-artificial-intelligence.html?module=inline) on the use of facial recognition in machine learning.
 
-###  Recap  
+### Recap
 
-The data we use to train our model affects the results of our model. This can lead to discriminatory practices. If our model under-represents or over-represents different groups that can bias our results.  Thus, it is important to evaluate the diversity in your dataset and evaluate our results on a diverse dataset and to keep in mind what we are using this model for.  
+The data we use to train our model affects the results of our model. This can lead to discriminatory practices. If our model under-represents or over-represents different groups that can bias our results. Thus, it is important to evaluate the diversity in your dataset and evaluate our results on a diverse dataset and to keep in mind what we are using this model for.
```

Comment:
"This can lead to discriminatory practices. If our model under-represents or over-represents different groups that can bias our results" - seems that they should be one sentence instead of two.